nix/overlays/nixpkgs-overrides.nix

```diff
@@ -1,7 +1,9 @@
 self: super: {
+  naersk = super.callPackage ((import ../sources.nix).naersk) { };
+  nixpkgs-fmt = super.callPackage ((import ../sources.nix).nixpkgs-fmt) { };
```

Comment:
Ideally we should put comments in here saying when we can go back to the nixpkgs version - what's wrong with the nixpkgs version of nixpkgs-fmt? Just that it's too old? Should we go back when we update to 20.09?

Reply:
yes it is unfortunately a bit old - upstream has already seen a 1.0 release that features some considerable changes in how the code is formatted.

I can (and probably should) add a comment though that we can switch to using nixpkgs-fmt from nixpkgs when switching to 20.09

Reply:
Comment added, thanks for pointing that out.